static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -0,0 +1,145 @@
+import {forwardRef as reactFowardRef, useEffect, useState} from 'react';
+
+import Input from 'sentry/components/forms/controls/input';
+import Slider from 'sentry/components/forms/controls/rangeSlider/slider';
+import SliderAndInputWrapper from 'sentry/components/forms/controls/rangeSlider/sliderAndInputWrapper';
+import SliderLabel from 'sentry/components/forms/controls/rangeSlider/sliderLabel';
+import type SliderProps from 'sentry/components/forms/controls/rangeSlider/sliderProps';
+import {t} from 'sentry/locale';
+import {defined} from 'sentry/utils';
+
+function RangeSlider({
+  value,
+  allowedValues,
+  showCustomInput,
+  name,
+  disabled,
+  placeholder,
+  formatLabel,
+  className,
+  onBlur,
+  onChange,
+  forwardRef,
+  showLabel = true,
+  ...props
+}: SliderProps) {
+  const [sliderValue, setSliderValue] = useState(
```

Comment:
By looking at the code, I think we could just drop this state entirely as it seems to be a derived value from value prop and allowedValues set

Reply:
This is existing code unrelated to the exports. Moving to new PR https://github.com/getsentry/sentry/pull/33219